client/lbclient/datasources.local.js

```diff
@@ -2,6 +2,6 @@ var GLOBAL_CONFIG = require('../../global-config');
 
 module.exports = {
   remote: {
-    url: GLOBAL_CONFIG.restApiUrl
+    url: 'http://localhost:3000' + GLOBAL_CONFIG.restApiUrl
```

Comment:
See https://github.com/strongloop/strong-remoting/issues/200

The host + port must not be hard-coded here. Check the way how GLOBAL_CONFIG is constructed and modify the code to produce a full URL, so that `datasources.local.js` can stay unchanged.